src/Workspaces/Core/Portable/Storage/SQLite/SQLitePersistentStorage.Accessor.cs

```diff
@@ -54,9 +38,9 @@ public Accessor(SQLitePersistentStorage storage)
             protected abstract void BindFirstParameter(SqlStatement statement, TDatabaseId dataId);
             protected abstract TWriteQueueKey GetWriteQueueKey(TKey key);
 
-            public async Task<Checksum> ReadChecksumAsync(TKey key, CancellationToken cancellationToken)
+            public Checksum ReadChecksum(TKey key, CancellationToken cancellationToken)
```

Comment:
note that reads/writes can now be synchronous.  This means we can actually use spans here in a future change (if we're willing to change the IPersistentStorageService API.